packages/app-clipper/popup/src/App.js

```diff
@@ -402,6 +404,19 @@ class AppComponent extends Component {
 			);
 		};
 
+		const openNewNoteButton = () => {
+
+			if (this.state.newNoteId === null) {
+				return undefined;
+			} else {
+				return (
+					// The jopin:// link must be opened in a new tab. When it's opened for  the first time, a system popup will ask for the user's permission.
+					// The popup is too small to show the permission and the user will not be able to see the buttons.
+					<a className="Button" href={`joplin://x-callback-url/openNote?id=${this.state.newNoteId}`} target="_blank"> Open newly created note</a>
```

Comment:
Thanks for the detailed explanation, it's always useful to have the reasoning for this kind of code.

Also please escape `this.state.newNoteId` as a query parameter (`encodeURIComponent`)